domain/entities/stock_item.py

```diff
@@ -2,25 +2,22 @@
 from datetime import datetime
 
 from domain.entities.base_entity import BaseEntity
+from domain.entities.stock_group import StockGroup
 from domain.entities.stock_level import StockLevel
 from domain.entities.stock_location import StockLocation
 
 
 @dataclass
 class StockItem(BaseEntity):
+    days_until_stocktake_alert: int = None # TODO: have a global fallback option if this is 0
```

Comment:
yeah that's right

just realised an issue though is the UI says "0 to disable feature" but i was also thinking of a fallback global option....hmmmmmmmmm need to think this over more

Reply:
Wdym by fallback global option?

Set it for all stock?

Reply:
Is this fine to resolve?

Reply:
right so when this feature is actually implemented completely it will be:

- I enable the alerts for this stock item either on create or later on (see new property I added)
- I leave it at zero so the global number is used in case you're lazy or want consistency, maybe we don't need this though?
- I can set the number of days before a stocktake alert is created for this stock item

after that is all the various logic bits for how alerts are going to work

Reply:
this is safe to resolve, i added various todos and feature points in the excel doc